Library/src/Web/HttpRequest.cpp

```diff
@@ -160,6 +160,30 @@ HttpProgress& HttpRequest::GetProgress()
 	return Progress;
 }
 
+bool ResultCodeValidForRetry(csp::web::EResponseCodes Status)
+{
+	if (Status == csp::web::EResponseCodes::ResponseTooManyRequests					 // 429
```

Comment:
We can simply return the conditional here instead of an actual if/else.

Also, is there a reason to check each 500 response? Why not something like:

```
return 
    Status == csp::web::EResponseCodes::ResponseTooManyRequests ||
    Status == csp::web::EResponseCodes::ResponseRequestTimeout ||
    Status  >= 500;
```

Reply:
Yeah I get that, but the cast would allow for us to avoid human error (missing codes etc) both now and in the future. Think its worth it in this instance.